### PR TITLE
add TypeScript hello world template w/ Lambda Powertools integration

### PR DESCRIPTION
**Pull request by bpauwels**
* feat: add TypeScript hello world template w/ Lambda Powertools for TypeScript integration
* Update manifest to include new workflow

### CODE REVIEW

**Comment by bpauwels** [comment]
as discussed with romboiko@;
Contact pauwelsb@ for questions

File: manifest.json [hunk 1]
```diff
@@ -323,6 +323,14 @@
       "packageType": "Zip",
       "useCaseName": "Hello World Example"
     },
+    {
+      "directory": "nodejs14.x/cookiecutter-aws-sam-hello-powertools-typescript-nodejs",
```
Comment (mndeveci) [inline manifest.json:327]: Can you move this information into `manifest-v2.json` file?

**Review by mndeveci — CHANGES_REQUESTED** [review]
Thanks for your contribution, can you move changes from `manifest.json` file into `manifest-v2.json` and some testing? Example testing for existing nodejs14 can be found here: https://github.com/aws/aws-sam-cli-app-templates/blob/master/tests/integration/unit_test/test_unit_test_nodejs14_x.py

Rest looks good to me.

File: manifest.json (hunk 1 above)
Comment (bpauwels) [reply manifest.json:327]: Hi, I've just done so! I see you have updates to node16.x - should I change this in my structure or is it done when merging?

**Review by mndeveci — APPROVED** [review]

**Pull request merged**